### PR TITLE
Update the psammead-locale package

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves chromaticqa failures on IE11 for https://github.com/bbc/simorgh/pull/3468 

**Overall change:** _updates the package.json._

**Code changes:**

- _Updates the package.json with a sideEffects flag of the module it is bundling.._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "name": "@bbc/psammead-locales",
   "version": "2.1.0",
   "description": "A collection of locale configs, used in BBC World Service sites",
+  "main": "dist/index.js",
```
Comment (dr3) [inline packages/utilities/psammead-locales/package.json]: when psammead is built these files dont exist :( Im not sure what the IE11 issue is, maybe the `"sideEffects": false,` fixes it? Not sure why though :(  Sorry I cant be more helpful

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-locales/package.json (hunk 1 above)
Comment (dr3) [reply packages/utilities/psammead-locales/package.json]: Happy to try merging just the `"sideEffects": false,` and see if that helps?

Comment (RayNjeri) [reply packages/utilities/psammead-locales/package.json]: yeah adding just this `"sideEffects": false` should fix it.
Testing it now, to be sure

**Review by thekp — APPROVED** [review]

File: packages/utilities/psammead-locales/package.json (hunk 1 above)
Comment (jamesdonoh) [reply packages/utilities/psammead-locales/package.json]: Does anyone know if there is a reason why sideEffects: false should not be specified for all packages?

Comment (RayNjeri) [reply packages/utilities/psammead-locales/package.json]: @dr3 adding just the `sideEffects: false` flag does the trick,
Tested it out locally

**Review by thekp — COMMENTED** [review]
You will have to bump the version and update the changelog 👀

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by jamesdonoh — APPROVED** [review]
Thanks to all who helped figure this one out.

**Review by rhenshaw56 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

File: packages/utilities/psammead-locales/package.json (hunk 1 above)
Comment (dr3) [reply packages/utilities/psammead-locales/package.json]: @jamesdonoh We only really added it where needed, alot of packages are accessed via a nested `package.json` so doesnt need it at the base `package.json`, we just missed it with moment

**Pull request merged**